src/time_zone_lookup.cc

```diff
@@ -37,7 +37,12 @@
 #include <sdkddkver.h>
 // Include only when the SDK is for Windows 10 (and later), and the binary is
 // targeted for Windows XP and later.
-#if defined(_WIN32_WINNT_WIN10) && (_WIN32_WINNT >= _WIN32_WINNT_WINXP)
+// Note: Windows SDK added windows.globalization.h file in 10.0.10240.0,
+//   but mingw added it in 10.0.22621.0, so the version is checked against
+//   10.0.22621.0 in mingw.
```

Comment:
None of these version numbers appear in the conditional, which makes it difficult to see what is being checked.  Should the note be phrased in terms of the symbols instead?

Reply:
Done.